appsec/src/extension/ddtrace.c

```diff
@@ -89,6 +91,12 @@ static void dd_trace_load_symbols(void)
             dlerror()); // NOLINT(concurrency-mt-unsafe)
     }
 
+    _ddtrace_ip_extraction_find = dlsym(handle, "ddtrace_ip_extraction_find");
```

Comment:
For the backlog but we're reaching the point where it would be useful to have some macros to do this and generate the relevant passthrough functions as well.